django_fsm/__init__.py

```diff
@@ -264,7 +264,10 @@ def deconstruct(self):
         return name, path, args, kwargs
 
     def get_state(self, instance):
-        return instance.__dict__[self.name]
+        if self.name in instance.__dict__:
+            return instance.__dict__[self.name]
+
+        return getattr(instance, self.name)
```

Comment:
Urgh, I wish GitHub would have notified me of that comment. We are mixing this library with `django-hvad`, which does some funky stuff under the hood. So in this case translated attributes are not in `__dict__`, but `getattr` succeeds (because the library proxies the getattr call to the underlying modelnameTranslation model, which actually contains the data).